editor/plugins/tile_set_editor_plugin.cpp

```diff
@@ -3055,7 +3055,7 @@ Vector2 TileSetEditor::snap_point(const Vector2 &point) {
 }
 
 void TileSetEditor::add_texture(Ref<Texture> p_texture) {
-	texture_list->add_item(p_texture->get_path().get_file());
+	texture_list->add_item(p_texture->get_path().get_basename().get_file());
```

Comment:
The order is wrong, it should be `get_file().get_basename()`. `get_basename()` gives you the filename without the extension.